core/jazz_assets/components/crud/getList.js

```diff
@@ -0,0 +1,94 @@
+ /**
+      Search a Asset-Catalog entry in dynamodb table
+      @module: getList.js
+      @description: CRUD functions for asset catalog
+      @author:
+      @version: 1.0
+  **/
+
+ const utils = require("../utils.js"); //Import the utils module.
+ const logger = require("../logger.js")(); //Import the logging module.
+
+ module.exports = (query, asset_table, onComplete) => {
```

Comment:
There is too much going on in this handler. Can you modularize the code (create some functions)? and make it more readable and testable?

Reply:
Now this is consistent across all services. The only logic it does is, create/format a get payload and hit the DB